python/helpers/persistent_storage.py

```
@@ -0,0 +1,158 @@
+        return {
+            "copied_files": self.copied_files,
+            "copied_dirs": self.copied_dirs,
+            "skipped_items": self.skipped_items,
+        }
+
+
+def _resolve_path(path: str) -> str:
+    return os.path.abspath(os.path.expanduser(path))
+
+
+def _iter_content_paths(base_dir: str, skip_default: bool = True) -> Iterable[Tuple[str, str]]:
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)